pkg/destroy/openstack/swift.go

```diff
@@ -1,43 +0,0 @@
-package openstack
-
-import (
-	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/containers"
-	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/objects"
-	"github.com/gophercloud/utils/openstack/clientconfig"
-)
-
-// DeleteSwiftContainer deletes a container and all of its objects.
-func DeleteSwiftContainer(name string, cloud string) error {
```

Comment:
It is strange that you can now safely remove this.  What if folks want to delete a cluster created by a previous installer (and so using Swift) which got a Swift bootstrap Ignition store, but then failed to install.  Wouldn't that leak the Swift container with `destroy cluster` not using this function?  Of course, the fact that nothing else in `pkg/destroy/openstack` calls this function means that leak is not *new* in this PR.  But it still seems like a leak to me.

Reply:
Never mind, @Fedosin points out that this function was pretty similar to [`deleteContainers`][1], which is left alone by this PR.

[1]: https://github.com/openshift/installer/blob/96d416ee36d8660c3f2cb33863587455278f4ed2/pkg/destroy/openstack/openstack.go#L541